addons/mail/static/src/new/core/channel_member_model.js

```diff
@@ -5,10 +5,22 @@
  * @typedef Data
  * @property {number} id
  * @property {number} partnerId
+ * @property {number} threadId
  */
 export class ChannelMember {
-    constructor(_state, { id, partnerId }) {
-        Object.assign(this, { _state, id, partnerId });
+    static insert(_state, data) {
```

Comment:
While you have diff on the line, I would name the param `state` for consistency with other insert methods.